intrahost.py

```diff
@@ -460,6 +460,21 @@ def merge_to_vcf(
                     the number of alignment files equals the number of chromosomes / segments
     '''
 
+    guessed_samples = []
+    isnvs_to_use = []
+    if not samples:
+        for isnvs_file in isnvs:
+            if os.path.getsize(isnvs_file) > 0:
+                isnvs_to_use.append(isnvs_file)
+                continue
+            for row in util.file.read_tabfile(isnvs_file):
+                guessed_sample_ID = sampleIDMatch(row[0])
```

Comment:
If I read this correctly, this code is unreachable, because anytime the `isnvs_file` is non-empty, the above `if` statement will `continue` and skip this `for` loop, but any time it's empty, the `for` loop won't do anything.

Reply:
Thanks, that was sloppy of me. I'll fix this and add a unit test where sample names are guessed so this has some test coverage.